.github/workflows/released.yml

```diff
@@ -0,0 +1,34 @@
+name: Merge trunk to develop branch and update tag
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  update:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.release.target_commitish }}
+
+      - name: Merge trunk to develop branch
+        uses: devmasx/merge-branch@v1.4.0
+        with:
+          type: now
+          from_branch: trunk
+          target_branch: develop
+          github_token: ${{ github.token }}
+
+      - name: Prepare "trunk_last_commit" variable
+        run: echo "trunk_last_commit=$(git rev-parse trunk)" >> $GITHUB_ENV
```

Comment:
Do we have to use this as an env variable instead of a variable within the pipeline?

Reply:
I got rid of the action and combined the tag push process into a single step.